samples/dnn/optical_flow.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+'''
+This sample using FlowNet v2 model to calculate optical flow.
+Original paper: https://arxiv.org/abs/1612.01925
+Original repo:  https://github.com/lmb-freiburg/flownet2
+To get original model:
+    wget --no-check-certificate https://lmb.informatik.uni-freiburg.de/resources/binaries/flownet2/flownet2-models.tar.gz
+    tar xvzf flownet2-models.tar.gz       
+
+To run model in OpenCV modify Flownetv2.prototxt.
+
+change ChannelNorm layers to series of layers:
```

Comment:
How many `ChannelNorm` layers inside? Maybe it make sense to put modified `.prototxt` to opencv_extra?

Reply:
I put modified .prototxt and .caffemodel to cloud.